cas_client/src/error.rs

```diff
@@ -63,6 +63,15 @@ pub enum CasClientError {
     PresignedUrlExpirationError,
 }
 
+impl From<reqwest::Error> for CasClientError {
+    fn from(value: reqwest::Error) -> Self {
+        // strip non-domain info from url, primarily query parameters which can be sensitive information
+        let domain = value.url().and_then(|url| url.domain()).unwrap_or_default().to_string();
```

Comment:
I would rather we strip out query parameters instead of keeping only the domain. Not having the path will make it difficult to consult CDN logs for problematic requests.